ramesh-portfolio/index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
Please don't delete the whole code and also add the website view screenshot image as well.

Reply:
Sure Prateek